api/kyverno/v2alpha1/global_context_entry_status.go

```diff
@@ -22,6 +22,9 @@ type GlobalContextEntryStatus struct {
 	Ready bool `json:"ready" yaml:"ready"`
 	// +optional
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	// Indicates the time when the globalcontextentry was last refreshed
```

Comment:
Suggest change to "Indicates the time when the globalcontextentry was last refreshed successfully for the API Call".